frontend/src/GameRouter.tsx

```diff
@@ -6,7 +6,9 @@ import {
   Redirect,
   useParams,
 } from "react-router-dom";
-import BasicPage from "./BasicPage";
+import BasicPage from "./pages/BasicPage";
```

Comment:
Could be useful to have an index file for pages which re-exports all the pages from a single file. It would condense the imports but it would also need to be updated everytime a page is created or deleted so there's a tradeoff

Reply:
Will do!